frontend/src/components/HospitalAdmin.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Admin, Resource} from 'react-admin';
+import jsonServerProvider from  'ra-data-json-server';
+import InventoryList from './InventoryList';
+import UserList from './UserList';
+import HospitalAdminLandingPage from './HospitalAdminLandingPage';
+import HospitalAddressShow from './HospitalAddressShow';
+import HospitalAddressEdit from './HospitalAddressEdit';
+import UserEdit from './UserEdit';
+import InventoryEdit from './InventoryEdit';
+
+export default () => {
+
+  // const dataProvider = jsonServerProvider('https://jsonplaceholder.typicode.com');
+  const dataProvider = jsonServerProvider('http://my-json-server.typicode.com/naijauser/infomedapi');
```

Comment:
Defining the data provider is an important thing here. Without it, this PR doesn't move us any closer to the finishing line.